packages/ui/components/avatar/Avatar.tsx

```diff
@@ -40,7 +40,7 @@ export function Avatar(props: AvatarProps) {
   let avatar = (
     <AvatarPrimitive.Root
       className={classNames(
-        "bg-emphasis item-center relative inline-flex aspect-square justify-center overflow-hidden rounded-full",
+        "bg-emphasis item-center relative aspect-square justify-center overflow-hidden rounded-full",
```

Comment:
This was making the avatar have a weird margin top, was there to better display an icon fallback which is now supported by the changes at the bottom